compute/resource_cluster.go

```diff
@@ -33,6 +33,21 @@ func ResourceCluster() *schema.Resource {
 	}
 }
 
+func wrapMavenExclusions(scm *schema.Schema) {
```

Comment:
misguiding function name - it's not wrapping, because it doesn't take and return something, it's "modifying" the pointer.

Reply:
done